task/sql-tasks.js

```diff
@@ -199,7 +310,23 @@ async function task_1_14(db) {
  *
  */
 async function task_1_15(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    select 
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 1)  as "January",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 2) as "February",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 3) as "March",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 4) as "April",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 5) as "May",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 6) as "June",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 7) as "July",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 8) as "August",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 9) as "September",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 10) as "October",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 11) as "November",
+(select count(*) from Orders where year(OrderDate) = 1997 and month(OrderDate) = 12) as "December"
+from Orders group by 1,2,3,4,5,6,7,8,9,10,11,12

```

Comment:
using group by doesn't make sense, if you do it like this you coud've just skip from-clause altogether. from clause is optional

Reply:
try to change this query to look like this 

select 
   count(<some expression) as "JAN",
   ......
FROM Orders
where year = 1997